Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -374,3 +373,10 @@ Multisite is a feature of Ceph that allows object stores to replicate its data o
 Multisite also allows object stores to be independent and isolated from other object stores in a cluster.
 
 For more information on multisite please read the [ceph multisite overview](ceph-object-multisite.md) for how to run it.
+
+## Using Swift and Keystone
+
+You can also access an object store using the [Swift API](https://developer.openstack.org/api-ref/object-store/index.html). 
```

Comment:
"you"

Reply:
https://github.com/rook/rook/pull/13807#discussion_r1557848447